Content.Client/Singularity/SingularityLevelChangedEvent.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Robust.Shared.GameObjects;
+
+namespace Content.Client.Singularity
+{
+    public class SingularityLevelChangedEvent : EntityEventArgs
```

Comment:
does this really need to be its own file? most other entityeventargs are just in the relevant file, and its not like this makes it more searchable or something

Reply:
True, though it was a leftover file anyway so it's now deleted. Fixed in https://github.com/space-wizards/space-station-14/pull/4194/commits/17db10fdaf0e540e0d58658f0565a51f5c89b3c4